### PR TITLE
feat: SURVEY-19184 to add new viewing list theme

### PR DESCRIPTION
**Pull request by flying3615**
image
Author Checklist

- [ ] appropriate description or links provided to provide context on the PR
- [ ] self reviewed, seems easy to understand and follow
- [ ] reasonable code test coverage
- [ ] change is documented in Storybook and/or markdown files

Reviewer Checklist

- Follows convention
- Does what the author says it will do
- Does not appear to cause side effects and breaking changes
  - if it does cause breaking changes, those are appropriately referenced

Post merge

- [ ] Post about the change in #lui-cop

Conventional Commit Cheat Sheet:
**build**: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
**ci**: Changes to our CI configuration files and scripts (example scopes: Circle, BrowserStack, SauceLabs)
**docs**: Documentation only changes
**feat**: A new feature
**fix**: A bug fix
**perf**: A code change that improves performance
**refactor**: A code change that neither fixes a bug nor adds a feature
**test**: Adding missing tests or correcting existing tests

### CODE REVIEW

**Review by egeofalcao — APPROVED** [review]

File: src/stories/grid/GridViewList.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        data-testid={"readonly"}
+        {...props}
+        selectable={false}
+        rowSelection={"single"}
+        externalSelectedItems={externalSelectedItems}
+        setExternalSelectedItems={setExternalSelectedItems}
+        columnDefs={columnDefs}
+        rowData={rowData}
+        sizeColumns={"fit"}
+        theme={"ag-theme-step-view-list-default"}
```
Comment (egeofalcao) [inline src/stories/grid/GridViewList.stories.tsx:117]: maybe the theme name could be "ag-theme-step-iceberg"

File: src/styles/GridTheme.scss [hunk 2]
```diff
@@ -249,3 +257,45 @@ $grid-base-font-size: calc(#{lui.$base-font-size} - 1px);
...
+  .ag-header-group-cell {
+    font-weight: normal;
+    font-size: 22px;
+  }
+
+  .ag-header-cell {
+    font-size: 14px;
+  }
+
+  .ag-cell {
```
Comment (matttdawson) [inline src/styles/GridTheme.scss:293]: This is controlled by cell-horizontal-border, is this needed?

File: src/styles/GridTheme.scss [hunk 3]
```diff
@@ -249,3 +257,45 @@ $grid-base-font-size: calc(#{lui.$base-font-size} - 1px);
...
+    }
+  }
+
+  .ag-header .ag-header-cell[aria-colindex="1"] {
+    padding-left: 32px;
+    padding-right: 12px;
+  }
+
+  .ag-header {
+    background: lui.$iceberg;
```
Comment (matttdawson) [inline src/styles/GridTheme.scss:281]: header-background-color ?

File: src/stories/grid/GridViewList.stories.tsx (hunk 1 above)
Comment (matttdawson) [reply src/stories/grid/GridViewList.stories.tsx:117]: Yeh, list-default not a good name.  This theme your making should apply to all of survey as Andi says it's the new theme.  So maybe we should just versioning it?  step-devault-v2 ?

**Pull request merged**